src/index.ts

```diff
@@ -49,7 +131,7 @@ function Upload (cfg) {
   this.userProject = cfg.userProject;
 
   if (cfg.key) {
-    const base64Key = Buffer.from(cfg.key).toString('base64');
+    const base64Key = Buffer.from(cfg.key as string).toString('base64');
```

Comment:
Why is the `as string` needed? `Buffer.from` can accept both `string` and `Buffer`.

Reply:
Well, this is a weird story.  
https://github.com/Microsoft/TypeScript/issues/23155

I added a healthy comment to the code to explain what's up.